crates/bevy_ecs/Cargo.toml

```diff
@@ -21,7 +21,7 @@ bevy_tasks = { path = "../bevy_tasks", version = "0.12.0" }
 bevy_utils = { path = "../bevy_utils", version = "0.12.0" }
 bevy_ecs_macros = { path = "macros", version = "0.12.0" }
 
-async-channel = "1.4"
+async-channel = "2"
```

Comment:
any reason not to specify 2.1.0 as minimum version? according to the changelog this bumps the future-lite version (https://github.com/smol-rs/async-channel/pull/70) to match the one bevy now exports (https://github.com/bevyengine/bevy/pull/10670)